veracruz-mcu-client/transport_protocol.proto

```diff
@@ -0,0 +1,259 @@
+//! Protocol buffers for Veracruz transport protocol messages
```

Comment:
This looks to be very similar to the proto file in the transport-protocol crate. Why is it ?mostly? duplicated here?

Reply:
Ah yes! This is a problem with the dockers that I'd be happy to have suggestions, I'm not sure what the correct solution is.

You are correct that veracruz-mcu-client/transport_protocol.proto is identical to transport_protocol/protos/transport_protocol.proto

I initially wanted to symlink these together, but veracruz-mcu-client currently builds in its own docker. So at build time the symlink points no where, because the destination doesn't exist in veracruz-mcu-client's docker.

I could add a step in the docker that copies transport_protocol.proto when you start the docker, but this option would mean that 1. transport_protocol.proto would not be updated unless you restarted your docker, 2. restarting docker would always trigger a rebuild due to the new timestamp (maybe there's a mv option to override this), 3. the source of transport_protocol.proto would be a bit hidden, maybe causing maintenance issues in the future.

Another option would be to change the veracruz-mcu-client docker to start from the Veracruz root directory, actually now that I'm typing this I don't see a reason not to, so that may be what to do

Reply:
`cp -p` should do this. However, is there a problem with just including the `veracruz-transport/` directory in the docker container?

Reply:
Sorry I forgot to update this one, I've added a step in the Makefile to copy the transport_protocol.proto file during `make docker`.

https://github.com/veracruz-project/veracruz/blob/e40344489c78019b35a2773a50ba1f19a60310bc/veracruz-mcu-client/Makefile#L42-L44

I tried adding a COPY for the file in the Dockerfile, but it didn't work as the file is outside docker's "build context"

I can see if there's a way to change the build context to include all of Veracruz and include the directory if we prefer that.